classes/Storage.php

```diff
@@ -126,10 +126,22 @@ public static function update($page, $uid, $data = [], $filename)
 
                 // if the data variable is an array, update the fields contained within (default is an empty array, hence no updates)
                 if (is_array($data)) {
+                    // depending on the entry type, certain fields can not be deleted
+                    if ($entry['type'] == 'comment') {
```

Comment:
When comparing variables, it’s generally more common in most standards to use the strict comparison operator `===`, because most of the time you are comparing values with each other, you know their datatype and want to clearly distinguish between values, such as `"0"` and `0`. It’s not only safer in some cases (e.g. [`strpos(…) !== false`](https://www.php.net/strpos), but also seems to be a bit faster. In this particular case, it does not really matter, but using it by default makes it easier not to forget it at important times.

Reply:
Thanks, will try to make that a habit :)